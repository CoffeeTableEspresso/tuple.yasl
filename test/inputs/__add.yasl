const tuple = requie('tuple')

const a = tuple(1)
const b = tuple(true, false)

echo a + b
echo b + a
echo a + a
echo b + tuple()
echo tuple() + b
echo tuple() + tuple()
