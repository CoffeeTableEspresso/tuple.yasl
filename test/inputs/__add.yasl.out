tuple(1, true, false)
tuple(true, false, 1)
tuple(true, false)
tuple(true, false)
tuple()

